### DISCUSSION TITLE
How to solve the issue where data fetched using useFetch isn’t requested again after refreshing the page?

### DISCUSSION THREAD

**Discussion by zsk-coder** [nuxt/nuxt#27878] [Questions]
After entering the page and completing the request, refreshing the page will not re-issue the request.  My demand is to obtain the server data in real time, so I do not want to go to the cache.  I have tried to add a time stamp as the key in the request, but my demand is still not met, please help me solve it, thank you very much

**Comment by oneminch** [comment]
The `options.cache` parameter of `useFetch` allows you to specify whether you want the request to be cached. You can either set that to a boolean value or a specific value like `no-cache` according to the [documentation](https://nuxt.com/docs/api/composables/use-fetch#params).

```js
const { data } = await useFetch('/api/endpoint', {
  cache: false // or "no-cache"
})
```

Alternatively, you can use the `refresh` function that is returned from `useFetch` and call it every time the page is mounted.

```js
import { onMounted } from 'vue'

const { data, refresh } = await useFetch('/api/endpoint')

onMounted(() => {
  refresh()
})
```
